hypha/settings/base.py

```diff
@@ -20,7 +20,8 @@
 # Hypha custom settings
 
 # Set the currency symbol to be used.
-CURRENCY_SYMBOL = env.str('CURRENCY_SYMBOL', '$')
+CURRENCY_CODE = env.str('CURRENCY_CODE', 'USD')
+LOCALE = env.str('LOCALE', 'en_US')
```

Comment:
If this is only used for currency should we call it `CURRENCY_LOCALE`?

Reply:
Yeah, I think for other places we can use LANGUAGE_CODE only currency needs to be strict so I'll rename it.

Reply:
Or can't we just use LANGUAGE_CODE for currency as well(as @theskumar suggests)? It is also a similar setting and fixed for an organization. Or it might create any difference/issues?

Reply:
@frjo @sandeepsajan0 please take a look at my comment here: https://github.com/HyphaApp/hypha/issues/3448#issuecomment-1607165356